CLUSTER.md

```diff
@@ -0,0 +1,31 @@
+# GTSFM on CLUSTER
+
+### How to Run GTSfM on a Cluster?
+
+GTSfM uses [SSHCluster](https://docs.dask.org/en/stable/deploying-ssh.html#dask.distributed.SSHCluster) module of [Dask.distributed](https://distributed.dask.org/en/stable/) library to allow utilizing clusters for GTSfM runs. This is a step-by-step guide on how to set up your machines for a successful run on a cluster.
+
+1. Enable passwordless SSH between all the workers on the cluster
+    - Log in into a machine
+    - For each of the other workers on the cluster run
+        * ssh-copy-id
```

Comment:
could we put this in code quotes? Do we mean
`ssh-copy-id username@machine_ip_address_of_another_worker`?

Reply:
(the specific bash commands here weren't quite clear to me)

Reply:
Done.

Reply:
Resolved, thanks!